lib/enum/cardinalDirection.js

```diff
@@ -1,8 +1,8 @@
 var CardinalDirection = {
-  NORTH: 'N',
-  SOUTH: 'S',
-  EAST: 'E',
-  WEST: 'W'
+  NORTH: "N",
```

Comment:
I'll put everything back to single quotes. Somehow I managed to miss that part of the style guide :p